tests/modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ASHLAR } from '../../../../../modules/modules/nf-core/ashlar/main.nf'
+// we zero out the UUID of output tiff images with ZERO_UUID so we get a consistent md5sum
+include { ZERO_UUID } from './zero_uuid.nf'
```

Comment:
I think you forgot to `git add zero_uuid.nf` -- I don't see it here?

Reply:
You're right, I forgot to add it.  Thanks!
It's in there now.  Please take a look when you've got a chance.